### PR TITLE
chore: remove what's new functionality

### PR DESCRIPTION
**Pull request by mhatvan**
#### Description of Change
- remove code related to announcements
- change AppUpdateInfoBar to point to GitHub changelog instead of what's new page
- remove what's new menu entry
- disable react/jsx-one-expression-per-line conflicting with prettier

#### Motivation and Context
Closes https://github.com/getferdi/ferdi/issues/1822

We decided to remove the What's new menu entry, since it essentially does the same thing as displaying the changelog in GitHub. This is handled by the "Changelog" menu entry in Ferdi.

I didn't remove the translation messages in each language, since this would cause a lot of file changes.

#### Checklist

- [x] My pull request is properly named
- [x] The changes respect the code style of the project (`npm run prepare-code`)
- [x] `npm test` passes
- [x] I tested/previewed my changes locally

### CODE REVIEW

File: src/components/AppUpdateInfoBar.js [hunk 1]
```diff
@@ -51,12 +43,16 @@ class AppUpdateInfoBar extends Component {
...
           type="button"
-          onClick={() => announcementActions.show({ targetVersion: nextAppReleaseVersion })}
+          onClick={() =>
+            openExternalUrl(
+              `${GITHUB_FERDI_URL}/ferdi/blob/develop/CHANGELOG.md`,
+              true,
+            )
+          }
         >
           <u>{intl.formatMessage(messages.changelog)}</u>
```
Comment (vraravam) [inline src/components/AppUpdateInfoBar.js:57]: Do we want to remove the 'changelog' menu item that I had introduced?

File: src/lib/Menu.js [hunk 2]
```diff
@@ -513,44 +523,49 @@ const _titleBarTemplateFactory = (intl, locked) => [
     submenu: [
       {
         label: intl.formatMessage(menuItems.learnMore),
-        click() { openExternalUrl(LIVE_API_FERDI_WEBSITE, true); },
+        click() {
+          openExternalUrl(LIVE_API_FERDI_WEBSITE, true);
+        },
       },
       {
         label: intl.formatMessage(menuItems.changelog),
```
Comment (vraravam) [inline src/lib/Menu.js:531]: This might be collapsed into the "What's new"?

**Review by vraravam — CHANGES_REQUESTED** [review]

File: src/components/AppUpdateInfoBar.js (hunk 1 above)
Comment (mhatvan) [reply src/components/AppUpdateInfoBar.js:57]: I was working according to this comment of yours: https://github.com/getferdi/ferdi/issues/1822#issuecomment-903382206

So keeping `Changelog` and removing `Whats new`

File: src/lib/Menu.js (hunk 2 above)
Comment (mhatvan) [reply src/lib/Menu.js:531]: I kept the naming as it was before. There was no prior discussion of renaming `Changelog` to `What's new`

**Review by vraravam — APPROVED** [review]

**Comment by vraravam** [comment]
The only reason for keeping `What's new` and removing `Changelog` would be that we could reuse the i18n translations as-is.

**Pull request merged**